src/main/java/com/paneedah/weaponlib/ServerEventHandler.java

```diff
@@ -253,7 +261,7 @@ protected void onLivingHurtEvent(LivingHurtEvent livingHurtEvent) {
         		Vec3d eyes = livingHurtEvent.getEntityLiving().getPositionEyes(1.0f);
             	if(hit.hitVec.distanceTo(eyes) < 0.6f) {
             		
-            		//tSystem.out.println("Current headshot multiplier is " + BalancePackManager.getHeadshotMultiplier());
+            		//System.out.println("Current headshot multiplier is " + BalancePackManager.getHeadshotMultiplier());
```

Comment:
It is preferable to just delete this, anyway thanks to GitHub if we ever need it we can just see it in the history.